src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -1266,5 +1266,10 @@ public static void InitHandle(SafeHandle safeHandle, IntPtr handle)
             // To help maximize performance of P/Invokes, don't check if safeHandle is null.
             safeHandle.SetHandle(handle);
         }
+
+        public static int GetLastWin32Error()
+        {
+            return GetLastPInvokeError();
+        }
```

Comment:
So did we introduce GetLastPInvokeError just to be able to give a different name than GetLastWin32Error?

Reply:
So it seems the answer to my question is "yes", but the issue talks about obsoleting GetLastWin32Error, and I don't see that here.
```
[Obsolete("Use GetLastSystemError() or GetLastPInvokeError()", DiagnosticId="<NextID>", UrlFormat="<the proper URL>")]
```

Reply:
(Nevermind on the obsolete, I read the rest of the issue comments.)

Reply:
Yes. See the API review discussion notes at https://github.com/dotnet/runtime/issues/46843#issuecomment-767762870 . The original API proposal was to just add `SetLastWin32Error`.